docs/sources/developers/plugins/migration-guide/angular-react/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+description: Guide for migrating plugins from AngularJS to React
+keywords:
+  - grafana
+  - plugins
+  - migration
+  - plugin
+  - documentation
+title: Migrating plugins from AngularJS to React
+menutitle: Angular to React
+weight: 1000
+---
+
+# Migrate a plugin from AngularJS to React
+
+If you’re looking to migrate a plugin to the new plugin platform, then we recommend that you release it under a new major version. Consider keeping a release branch for the previous version to be able to roll out patch releases for versions prior to Grafana 7.
+
+While there's no 1-to-1 migration path from an Angular plugin to the new React platform, from early adopters, we’ve learned that one of the easiest ways to migrate is to:
+
+1. Create a new branch called `migrate-to-react`.
+1. Start from scratch with one of the templates provided by Grafana Toolkit.
```

Comment:
@leventebalogh just checking - are we keeping restructuring and updates in separate PRs? I'd expect we need to update the content away from Toolkit etc

Reply:
Yeah, I would ideally keep this one only for the restructuring (and for small changes that are necessary for the restructure, e.g. the redirect logic.)